autotest.pm

```diff
@@ -147,13 +142,13 @@ sub runalltests {
                 }
             }
             else {
-                if ($snapshots_supported && $flags->{milestone}) {
+                if ($snapshots_supported && ($flags->{milestone} || $bmwqemu::vars{MAKETESTSNAPSHOTS})) {
                     make_snapshot('lastgood');
```

Comment:
How do you jump to the latest failed test if the test before that didn't create a snapshot?